colorcoder/MinorColor.java

```diff
@@ -0,0 +1,37 @@
+package colorcoder;
+
+public enum MinorColor implements IColor{
```

Comment:
instead of Interface ,use Inheritance

Reply:
The base repo used enum, which does not allow inheritance, only interface implementation, this was making it difficult to refactor, in order to stick to the initial code that was provided

Reply:
Is there a better way to split it up?